enterprise/reporting/reporting.go

```diff
@@ -139,6 +139,12 @@ func (r *DefaultReporter) DatabaseSyncer(c types.SyncerConfig) types.ReportingSy
 	if err != nil {
 		panic(fmt.Errorf("could not run reports migrations: %w", err))
 	}
+	if config.IsSet("Reporting.autoVacuumCostLimit") {
+		_, err = dbHandle.ExecContext(r.ctx, fmt.Sprintf("ALTER TABLE reports SET (autovacuum_vacuum_cost_limit = %d);", config.MustGetInt("Reporting.autoVacuumCostLimit")))
+		if err != nil {
+			panic(err)
+		}
+	}
```

Comment:
Do we need this to be configurable?

Reply:
If we want this to run always we can create a new reports_always folder and run the migrator with `RunAlways: true`

Reply:
Any reason why you think the mentioned change is a better way to go? 🤔

Reply:
Next time we need to add another change that needs to run always we'll just need to prepare a new file, not another if statement

Reply:
Makes sense. Made the change